internal/provider/kubernetes/routes.go

```diff
@@ -274,6 +273,11 @@ func (r *gatewayAPIReconciler) processHTTPRoutes(ctx context.Context, gatewayNam
 							"name", refGrant.Name)
 					}
 				}
+
+				//targetBackend := gatewayapi.GetTargetBackendReference(backendRef.BackendRef, gatewayapi.NamespaceDerefOr(gatewayapi.NamespacePtr(httpRoute.Namespace), "default"))
```

Comment:
does this commented code need to be removed ?

Reply:
removed